drivers/volume/portworx/portworx.go

```diff
@@ -181,6 +193,25 @@ func (d *portworx) updateNode(n node.Node, pxNodes []api.Node) error {
 	return fmt.Errorf("failed to find px node for node: %v PX nodes: %v", n, pxNodes)
 }
 
+func (d *portworx) isMetadataNode(node node.Node, address string) bool {
+	members, err := d.getKvdbMembers(node)
+	ipRegex := regexp.MustCompile(`http:\/\/(?P<address>.*)\:\d+`)
+	if err != nil {
+		logrus.Errorf("Failed to get metadata nodes. Cause: %v", err)
+		return false
```

Comment:
we should not be ignoring the error here. failure to determine it's a kvdb node does not mean the node is not a metadata node.